docs/src/environments/restore.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Restore an environment from a backup
+sidebarTitle: Restore environments
+description: See how to restore an environment from a previous state.
+weight: -10
+---
+
+Once you have [backups of your environment](./backup.md), you might want to restore data from a previous point.
+
+To restore an environment, you need an [Admin role for that environment type](../administration/users.md).
+
+## 1. List available backups
+
+To restore a backup, first select one of the available backups.
+
+{{< codetabs >}}
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+Run the following command:
+
+```bash
+platform backups
+```
+
+You get a response similar to the following:
+
+```bash
+Backups on the project My Project (1234567abcdef), environment main (type: production):
++---------------------------+----------------------------+------------+
+| Created                   | Backup ID                  | Restorable |
++---------------------------+----------------------------+------------+
+| 2022-08-15T09:48:58+01:00 | 5ouvtgo4v75axijww7sqnftste | true       |
+| 2022-07-09T14:17:17+01:00 | 7jks7dru5xpx5p5id5wtypur2y | true       |
+| 2022-06-22T18:33:29+01:00 | f3jbyxlhtmalco67fmfoxs7n4m | true       |
++---------------------------+----------------------------+------------+
+```
+
+Select one of the backups marked as **Restorable** and copy its **Backup ID**.
+
+<--->
+---
+title=In the Console
+file=none
+highlight=false
+---
+
+- Navigate to the environment where you want to see backups.
+- Click **Backups**.
+
+Select one of the backups marked as having completed successfully {{< icon success >}}.
+
+{{< /codetabs >}}
+
+## 2. Restore from a backup
+
+To restore the backup you've selected, follow these steps:
+
+{{< codetabs >}}
+---
+title=Using the CLI
+file=none
+highlight=false
+---
+
+1. Run the following command:
+   
+   <!-- This is in HTML to get the variable shortcode to work properly -->
+   <div class="highlight">
+     <pre class="chroma"><code class="language-bash" data-lang="bash">platform backup:restore {{< variable "BACKUP_ID" >}}</code></pre>
+   </div>
+1. Press `enter` to agree with the consequences and continue.
```

Comment:
It's a destructive action, so it should be a little daunting, I think. Data that currently exists gets deleted, so people need to know there are consequences.

Separately, I wouldn't use `clear` in this context. And I'd put the purpose at the start of the sentence: https://developers.google.com/style/clause-order

Reply:
I think the reason this caught my attention is because I'm used to relying on formatted warnings for that kind of sensitive information. For me, it's always been: "either this is critical information and therefore it needs to stand out through CSS, or it's not that critical and let's not make our readers anxious about it". But I see you're not using formatted warnings or notes in the docs much? 

(And I agree with your other comments, that was definitely me struggling to find another way to put it!)

Reply:
We do [use notes/callouts](https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#use-notes-appropriately) although there is some [evidence people are _more_ likely to ignore them](https://www.nngroup.com/articles/tunnel-vision-and-selective-attention/). It's already an extra step in the procedure with its own warnings, though, so I'm not sure it's needed here. The use of `consequences` was an attempt at a sort of middle ground

Reply:
Ok, it makes sense!